app/src/links.js

```diff
@@ -35,14 +35,16 @@ module.exports = {
     resetAllowanceInfo: 'https://github.com/ethereum/EIPs/issues/20#issuecomment-263524729',
     // ---->
     blog: 'https://medium.com/streamrblog',
-    main: '/',
-    marketplace: '/marketplace',
+    root: '/',
```

Comment:
We should consolidate the two ways of importing routes we have now.

Reply:
I agree. I'd prefer to separate this task and consult with mister @mondoreale on how to best move forward there. 
Ticket for when that day comes:
https://streamr.atlassian.net/browse/PLATFORM-334